x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -201,7 +201,7 @@ export function ServiceOverviewTransactionsTable(props: Props) {
             color="euiColorVis1"
             compact
             series={latency.timeseries ?? undefined}
-            valueLabel={asDuration(latency.value)}
+            valueLabel={asMillisecondDuration(latency?.value)}
```

Comment:
Where was `latency` changed to optional?

Reply:
https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/server/lib/services/get_service_instances/index.ts#L34 the `joinByKey` return type makes properties optional.

Reply:
ah this is only the case on the instances table, the transaction table doesn't have this as an optional property. I'll go ahead and remove the optional chaining.